modules/navier_stokes/src/fvkernels/INSFVTKESourceSink.C

```diff
@@ -160,10 +160,11 @@ INSFVTKESourceSink::computeQpResidual()
                                    std::pow(_var(elem_arg, old_state), 0.5) /
                                    (NS::von_karman_constant * distance_vec[i]) / tot_weight;
 
+      // Additional 0-value terms to make sure new derivative entries are not added during the solve
       if (y_plus < 11.25)
-        destruction += destruction_visc;
+        destruction += destruction_visc + 0 * destruction_log;
       else
-        destruction += destruction_log;
+        destruction += destruction_log + 0 * destruction_visc;
```

Comment:
Please note that this is adding unneessary computational burden to the SIMPLE-like solvers. Please add a switch to identify if we are trying to solve these equations with a simple solve. If we are, please don't do the multiplication by 0.